arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -698,6 +698,29 @@ def rule_gfm_1_3_10(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_3_16(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.3.16] Remove the duplicate link:arcroleType element.
+    """
+    for modelArcRoleTypes in val.modelXbrl.arcroleTypes.values():
+        if modelArcRoleTypes and len(modelArcRoleTypes) > 1:
```

Comment:
I think this is sufficient:
```suggestion
        if len(modelArcRoleTypes) > 1:
```